src/App.css

```
@@ -329,8 +331,10 @@ img.warning-icon {
 	text-overflow: ellipsis;
 }
 
-#new-message-container > button {
-	display: grid;
+#new-message-container > button, .search-container > button {
```

Please avoiding use it, always prefer using classes.

```suggestion
.new-message-container {
```

Recommendation:
Learn how to use [BEM methodology](http://getbem.com/introduction/)